apps/nextjs/src/components/DialogControl/ContentOptions/AdditionalMaterialsStartAgain.tsx

```diff
@@ -0,0 +1,78 @@
+import { resourceTypesConfig } from "@oakai/additional-materials/src/documents/additionalMaterials/resourceTypes";
+
+import {
+  OakFlex,
+  OakHeading,
+  OakP,
+  OakPrimaryButton,
+  OakSecondaryLink,
+} from "@oaknational/oak-components";
+import invariant from "tiny-invariant";
+
+import {
+  useResourcesActions,
+  useResourcesStore,
+} from "@/stores/ResourcesStoreProvider";
+import { docTypeSelector } from "@/stores/resourcesStore/selectors";
+
+type AdditionalMaterialsStartAgainProps = {
+  closeDialog: () => void;
+};
+
+const AdditionalMaterialsStartAgain = ({
+  closeDialog,
+}: Readonly<AdditionalMaterialsStartAgainProps>) => {
+  const docType = useResourcesStore(docTypeSelector);
+  invariant(docType, "docType must be defined");
+  const docTypeDisplayName =
+    resourceTypesConfig[docType].displayName.toLowerCase();
+  const { resetToDefault } = useResourcesActions();
+  return (
+    <OakFlex
+      data-testid="chat-share-dialog"
```

Comment:
Seems this testid has been duplicated elsewhere but should really just be unique for the test in question